release/sdk-config.yaml.tmpl

```diff
@@ -40,6 +40,10 @@ commands:
   path: daml-helper/daml-helper
   desc: "Start the extractor"
   args: ["run-jar", "extractor/extractor.jar", "--"]
+- name: ledger-api-test-tool
```

Comment:
Could you explain why this should be shipped as part of the SDK release tarball and as part of the assistant? It doesn’t seem like something that a typical SDK user would want to use.

Reply:
What's the downside of shipping it as part of the SDK tarball? Just space, or anything else?

Reply:
I’m not really worried about space. I’m worried about confusing users by adding more commands and about complicating our test process or even worse, shipping untested things as part of the SDK tarball.

Reply:
I agree with the confusion argument. As discussed on Slack, this tool should be released over Maven coordinates only.

@gleber you OK with that?

Reply:
Dropped this